samples/bluetooth/iso_broadcast/prj.conf

```diff
@@ -4,9 +4,7 @@ CONFIG_BT_DEBUG_LOG=y
 CONFIG_BT_DEVICE_NAME="Test ISO Broadcaster"
 
 CONFIG_BT_ISO_MAX_CHAN=2
-CONFIG_BT_ISO_TX_BUF_COUNT=2
+CONFIG_BT_ISO_TX_BUF_COUNT=4
```

Comment:
Should `BT_CTLR_ISO_TX_BUFFERS` inherit the value of `BT_ISO_TX_BUF_COUNT` by default?